client/src/styles/config.css

```diff
@@ -62,5 +62,5 @@
   --q-link-color: var(--q-color-primary-5);
 
   /* layout */
-  --q-space-base: 16px;
+  --q-space-base: 8px;
```

Comment:
This should be either kept at 16px or the calculation needs to be adjusted everywhere where this is used.

Reply:
As we discussed I moved back space-base back to 16px -> https://github.com/nzzdev/Q-editor/pull/116/commits/33610ad844819d195f60a4841c18d6049b313742